health/rpc/Makefile

```diff
@@ -0,0 +1,16 @@
+PB = $(wildcard *.proto)
+GO = $(PB:.proto=.pb.go)
+
+all: $(GO)
+
+%.pb.go: %.proto
+	protoc -I=. -I=$(GOPATH)/src \
+	--go_out=\
+	plugins=grpc:\
+	. $<
+
+clean:
+	rm -f *.pb.go
+	rm -f *pb_test.go
+
+.PHONY: clean
```

Comment:
Lots of these Makefiles copied around the repo now. The syntax of this thing is a mystery to me, need to learn how they work, but is there a way to parameterize it so we can just have one and pass in the path to the dir we want to run on?

Reply:
Sure, it sounds like a good idea.
I'd even prefer a more general one, like a `make protos` in the root Makefile that will generate all protos recursively in the repo. Even if you're interested in a single one that changed, it's usually quite fast to worry about targeting a single one.

Reply:
I like that idea. I have a lot of learning to do about Makefiles, so for now I'll make an issue for it.